compiler/one-cmds/one-quantize

```diff
@@ -247,8 +309,12 @@ def main():
     # verify arguments
     _verify_arg(parser, args)
 
-    # quantize
-    _quantize(args)
+    if _utils._is_valid_attr(args, 'force_quantparam'):
+        # write quantization parameters
+        _write_qparam(args)
+    else:
+        # quantize
+        _quantize(args)
```

Comment:
How about insert this change into `_quntize` method?
```python
def _quantize(args):
    # get file path to log
    dir_path = os.path.dirname(os.path.realpath(__file__))
    logfile_path = os.path.realpath(args.output_path) + '.log'

	if _utils._is_valid_attr(args, 'force_quantparam'):
        # write quantization parameters
        _write_qparam(args)
		return
```

Reply:
Updated.